source/_components/sensor.otp.markdown

```diff
@@ -38,7 +38,11 @@ $ pip install pyotp
 $ python -c 'import pyotp; print("Token: " + pyotp.random_base32())'
 Token: IHEDPEBEVA2WVHB7
 ```
-
+To run in a docker container
+```shell
+docker exec -it <container name> python -c 'import pyotp; print("Token: " + pyotp.random_base32())'
```

Comment:
@balloob What do you mean by Home Assistant docker name? The name of the docker image (homeassistant/home-assistant) or the name of the containter (home-assistant)?

I put it as <container name> incase someone had renamed the container. My guess is you want "home-assistant" as per the instructions, which I'm happy to do.

Pulled both values from instructions here:
https://home-assistant.io/docs/installation/docker/